src/panels/config/scene/ha-scene-editor.ts

```diff
@@ -827,7 +844,7 @@ export class HaSceneEditor extends SubscribeMixin(
       }
 
       output[entityReg.entity_id] = {
-        entity_only: !this._devices.includes(entityReg.device_id!),
+        entity_only: this._single_entities.includes(entityReg.entity_id),
```

Comment:
Instead of going over `this._entityRegistryEntries`, go over `this._single_entities`